### PR TITLE
Prevent dropdown to close on click if muti select is true

### PR DESCRIPTION
**Pull request by haseeb1009**
Description of changes
Prevent dropdown to close on click if muti select is true.

Fixes [Issue # 1237](https://github.com/OperationCode/front-end/issues/1237)

### CODE REVIEW

**Comment by kylemh** [comment]
It doesn't look like it worked.

**Comment by haseeb1009** [comment]
@kylemh  Its working now, please check.
Operation Code _ Code Schools

**Comment by kylemh** [comment]
#### 📝Changed routes:


| [**/code_schools**](https://storybook-git-fork-haseeb1009-main.operation-code.vercel.app/code_schools) |
|:-:|
|<a href="https://storybook-git-fork-haseeb1009-main.operation-code.vercel.app/code_schools">Screenshot of /code_schools</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/MNbOuDt_p89MoaQ4QZSspIktX3U_.png">(view full size)</a> |







Commit f735b592567d54df6fdd06e165a9bf7acf7fee45 (https://storybook-git-fork-haseeb1009-main.operation-code.vercel.app).

**Comment by haseeb1009** [comment]
@kylemh Sorry, I did not get your last comment

File: pages/code_schools.js [hunk 1]
```diff
@@ -232,6 +232,7 @@ function CodeSchools({ allSchools }) {
               placeholder="Start typing a state..."
               className={styles.select}
               isMulti
+              closeMenuOnSelect={false}
```
Comment (kylemh) [inline pages/code_schools.js:235]: Why is this necessary? If `isMulti` is true, shouldn't it propagate down to ThemedReactSelect?

**Review by kylemh — APPROVED** [review]
Thanks so much for your 🎃 Hacktoberfest🎃&nbsp; contribution!

If you weren't aware, we're offering prizes too! You can read about the details [here](https://github.com/OperationCode/START_HERE#-hacktoberfest-).

Regardless, you've contributed, and we recommend you set yourself a reminder to submit the aforementioned form after November 1st (or after you meet all the criteria).

**Comment by haseeb1009** [comment]
> Thanks so much for your 🎃 Hacktoberfest🎃  contribution!
> 
> If you weren't aware, we're offering prizes too! You can read about the details [here](https://github.com/OperationCode/START_HERE#-hacktoberfest-).
> 
> Regardless, you've contributed, and we recommend you set yourself a reminder to submit the aforementioned form after November 1st (or after you meet all the criteria).

@kylemh  Thanks for approving the PR, do have a link where all those projects are listed?

**Comment by kylemh** [comment]
@haseeb1009 I dont understand your question

**Comment by haseeb1009** [comment]
> @haseeb1009 I dont understand your question

My bad, I found the repositories in [this link](https://github.com/OperationCode/START_HERE#-hacktoberfest-) that you posted above. Thank you for sharing.

**Comment by haseeb1009** [comment]
@kylemh  Can you merge the PR now?

**Comment by kylemh** [comment]
@haseeb1009 hopefully integration tests pass this time! I think we reached the rate limit on our resources API this weekend which is what prevented this from getting merged.

**Comment by kylemh** [comment]
It looks like there's an integration test that's correctly failing! @haseeb1009

**Comment by kylemh** [comment]
Closing in favor of https://github.com/OperationCode/front-end/pull/1238 but this PR will still count towards Hacktoberfest! Just not Operation Code's specific Hacktoberfest prizes.
